triple-lift.js

```diff
@@ -231,60 +250,81 @@ window.headertag.partnerScopes.push(function() {
 
         //? }
 
-        var yourBidder = new Partner(config);
-        window.headertag[PARTNER_ID] = {};
-        window.headertag[PARTNER_ID].callback = yourBidder.responseCallback;
-        window.headertag[PARTNER_ID].render = yourBidder.renderAd;
+        var tplBidder = new Partner(config);
+
+        try {
+            window.headertag[PARTNER_ID] = {};
+            window.headertag[PARTNER_ID].callback = tplBidder.responseCallback;
+            window.headertag[PARTNER_ID].render = tplBidder.renderAd;
+        } catch(e) {
+            console.log("Error: " + e);
+        }
 
-        callback(null, yourBidder);
+        callback(null, tplBidder);
     }
 
     function Partner(config) {
         var _this = this;
 
-        var __targetingType = config.targetingType;
-        var __supportedAnalytics = SUPPORTED_ANALYTICS;
-        var __supportedOptions = SUPPORTED_OPTIONS;
+        var targetingType = config.targetingType;
+        var supportedAnalytics = SUPPORTED_ANALYTICS;
+        var supportedOptions = SUPPORTED_OPTIONS;
 
-        var __creativeStore = {};
+        var prefetch = {
+            state: prefetchState.NEW,
+            correlator: null,
+            gCorrelator: null,
+            slotIds: [],
+            callbacks: []
+        };
+
+        var demandStore = {};
+        var creativeStore = {};
+        var demandObj = {
+            slot : {}
+        };
 
         /* =============================================================================
          * Set default targeting keys to be used for DFP. Values for omKey and idKey are
-         * mandatory. pmKey/pmidKey(deals) is only necessary if the partner will use a private market.
+         * mandatory. pmKey is only necessary if the partner will use a private market
+         * (deals).
          *
          * Standard values are:
          *
-         * omKey: ix_(PARTNER ID)_cpm
-         * pmKey: ix_(PARTNER ID)_cpm
+         * omKey: ix_(PARTNER ID)_om
+         * pmKey: ix_(PARTNER ID)_pm
          * idKey: ix_(PARTNER ID)_id
-         * pmidKey: ix_(PARTNER ID)_dealid
          */
-        var __targetingKeys = {
-            omKey: 'ix_tpl_cpm',
-            pmKey: 'ix_tpl_cpm',
-            idKey: 'ix_tpl_id',
-            pmidKey: 'ix_tpl_dealid'
+        // var omKey = 'ix_' + PARTNER_ID + '_om',
+        //     pmKey = 'ix_' + PARTNER_ID + '_pm',
+        //     idKey = 'ix_' + PARTNER_ID + '_id';
+
+        // TODO: change the middle set to use CONSTANTs;
+        var omKey = 'ix_tpl_om',
```

Comment:
Adding these keys seems to cause a failure in fetch from DFP -- can you validate that the headerbiddingtag.js / DFP are configured for these new keys to work appropriately?